src/Models/CompanyRole.php

```diff
@@ -62,7 +62,7 @@ public function __construct(Client $client, array $role)
 
         $this->id = $this->getValue('id');
         $this->name = $this->getValue('name');
-        $this->order = $this->getValue('order');
+        $this->order = $this->getValue('order', null);
```

Comment:
The second parameter is `null` by default. This implicit value is unnecessary and should be removed.